database/migrations/2015_10_25_125123_add_talk_revision_slides.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddTalkRevisionSlides extends Migration
```

Comment:
This isn't the case for existing database migrations, so I'm assuming it can be ignored.